legend-engine-testable/src/main/java/org/finos/legend/engine/testable/assertion/TestAssertionHelper.java

```diff
@@ -30,7 +32,10 @@
 {
     public static ObjectMapper buildObjectMapperForJSONComparison()
     {
-        return new ObjectMapper().enable(DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS);
+        return new ObjectMapper()
+                .enable(DeserializationFeature.USE_BIG_DECIMAL_FOR_FLOATS)
+                .enable(JsonGenerator.Feature.WRITE_BIGDECIMAL_AS_PLAIN)
```

Comment:
Do we really need this? Isn't setting node factory with the below statement enough?
If we need this can we have a test that fails without this statement, I believe the current test would pass even without this
If no, can we remove this and also the dependency that was added?

Reply:
These are needed. If we only keep `.setNodeFactory(JsonNodeFactory.withExactBigDecimals(true)`, `testEqualToJsonAssertionWithDecimalPrecision` will start failing. 
Please check my newly added tests.